project-management/src/main/java/life/qbic/projectmanagement/application/api/AsyncProjectService.java

```diff
@@ -222,6 +223,20 @@ Flux<SamplePreview> getSamplePreviews(String projectId, String experimentId, int
    */
   Mono<Sample> findSample(String projectId, String sampleId);
 
+  /**
+   * Submits multiple validation requests in a single service call.
+   *
+   * @param requests a list of one or more {@link ValidationRequest}.
+   * @return a {@link Flux} of {@link ValidationResponse}, providing the validation results for the
+   * submitted requests. Exceptions are provided as {@link Flux#error(Throwable)}.
+   * @throws UnknownRequestException if an unknown request has been used in the service call
+   * @throws RequestFailedException  if the request was not successfully executed
+   * @throws AccessDeniedException   if the user has insufficient rights
+   * @since 1.10.0
+   */
+  Flux<ValidationResponse> validate(List<ValidationRequest> requests);
```

Comment:
Wouldn't it make sense to allow for a `Flux<ValidationRequest>` as input here? This would allow the caller to choose when to emit requests. (and can also be a Flux.just call to wrap a list).

Reply:
So far we have not integrated Flux as parameter, and i wanted to keep it simple / in alignment with the API we have.

Reply:
I still think it makes sense here. In contrast to the experimental variables (where we want a all or nothing return), here we are fine with providing the requests one by one.